tests/Aspire.EndToEnd.Tests/.runsettings

```diff
@@ -1,26 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<RunSettings>
-  <xUnit>
-    <ShowLiveOutput>true</ShowLiveOutput>
-  </xUnit>
-  <RunConfiguration>
-    <!-- Timeout in ms, 15 minutes -->
-    <TestSessionTimeout>900000</TestSessionTimeout>
-    <!-- Filter out failing (wrong framework, platform, runtime or activeissue) tests -->
-    <TestCaseFilter>category!=failing</TestCaseFilter>
-  </RunConfiguration>
-  <LoggerRunSettings>
-    <Loggers>
-      <Logger friendlyName="trx">
-        <Configuration>
-          <LogFileName>TestResults.trx</LogFileName>
-        </Configuration>
-      </Logger>
-      <Logger friendlyName="console">
-        <Configuration>
-          <Verbosity>normal</Verbosity>
-        </Configuration>
-      </Logger>
-    </Loggers>
-  </LoggerRunSettings>
-</RunSettings>
+--show-live-output on
```

Comment:
if we wanted to experiment with parallel tests runs for a particular library, we'd put it in a file like this presumably? (right now none may be stable for parallel runs, nor diagnosable, but for traditional unit tests it may be possible)

Reply:
Either that, or add it to `TestingPlatformCommandLineArguments`.

Note: the current file is no longer really runsettings, it's a response file. RunSettings is actually not supported by core MTP (there is limited support for it through VSTestBridge, but xUnit doesn't use the bridge).

Reply:
You can also do `dotnet run --project path/to/test-project.csproj -- --help` and it will show you all the command-line options available.

Reply:
why do we still have .runsettings if it's not really supported?

Reply:
The file name is just `.runsettings` but it acts as a command-line response file. I can rename for clarity.

The original intent is very unclear to me, there are too many .runsettings scattered and it's not claer which is used for what and when.

Reply:
I cannot clarify. Possibly @radical can or perhaps it's just happenstance, this isn't a mature repo. Any cleanup welcome!

Reply:
Okay, I went ahead and cleaned up all the `.runsettings`.

Any customizations that are needed for a specific project can be done via `TestingPlatformCommandLineArguments` MSBuild property. This allows more flexibility as you can customize in `Directory.Build.[props|targets]` as well, based on any conditions. Previously, only one `.runsettings` file can be passed, and it was too hard to track which runsettings is used where.